go.mod

```diff
@@ -63,6 +63,7 @@ require (
 	golang.org/x/exp v0.0.0-20230713183714-613f0c0eb8a1
 	golang.org/x/sync v0.3.0
 	google.golang.org/grpc v1.57.0
+	gopkg.in/src-d/go-billy.v4 v4.3.2
```

Comment:
idk if that matters, but should you be using the go-git version?

https://github.com/src-d/go-billy#we-continue-the-development-at-go-gitgo-billy-this-repository-is-abandoned-and-no-further-updates-will-be-done-on-the-code-base-nor-issueprs-will-be-answered-or-attended